File: tools/archwiki/testdata/contract-multi/server/multi/multi.go
// Package multi holds two contracted symbols in one file.
package multi

// Full is complete: all four contract tags present.
// @contract
// @pre input positive
// @post result positive
// @err none
// @inv no side effects
func Full(input int) int { return input }

// Half is missing @err and @inv — must be caught even though Full is complete.
// @contract
// @pre input nonempty
// @post result joined
func Half(input string) string { return input }

File: tools/archwiki/testdata/contract-multi/app/app/src/main/java/dev/agentmirror/fixture/multi/Multi.kt
package dev.agentmirror.fixture.multi

/**
 * FullClient is complete: all four contract tags present.
 * @contract
 * @pre token non-null
 * @post connected
 * @err none
 * @inv session bounded
 */
object FullClient

/**
 * HalfClient is missing @post — must be caught even though FullClient is complete.
 * @contract
 * @pre token non-null
 * @err on bad token
 * @inv session closed on error
 */
object HalfClient
